bin/reth/src/node/mod.rs

```diff
@@ -95,6 +95,7 @@ impl Command {
             info!("Starting metrics endpoint at {}", listen_addr);
             prometheus_exporter::initialize(listen_addr)?;
             stages_metrics_describer::describe();
+            reth_transaction_pool::metrics::describe();
```

Comment:
We should standardize how this is going to be done, one uses a module the other uses a function. Perhaps we should just have a `X::describe()` on the metrics structs if we are using those? (they are not strictly necessary btw)

Reply:
I agree, I have changed the describers to be a method of each of metrics struct now and standardized the naming conventions. Also removed the `<crate>_metrics_describer` module